tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+envlist =
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-{sqlite,postgresql}
+    linters
+
+[testenv]
+deps =
+    .
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-postgresql: psycopg2-binary
+    factory_trytond
+setenv =
+    TESTPATH=tests
+    sqlite: TRYTOND_DATABASE_URI={env:SQLITE_URI:sqlite://}
+    postgresql: TRYTOND_DATABASE_URI={env:TRYTOND_DATABASE_URI:postgresql://}
+    TRYTOND_CONFIG = tests/etc/test.cfg
+
+passenv =
+basepython =
+    py3.5: python3.5
```

Comment:
Python-3.5 is already past end-of-life. I think it should be skipped for a just birthed software package.